packages/react-aria-components/src/Tabs.tsx

```diff
@@ -35,7 +35,11 @@ export interface TabListRenderProps {
    * The orientation of the tab list.
    * @selector [aria-orientation="horizontal | vertical"]
    */
-  orientation: Orientation
+  orientation: Orientation,
+  /**
+   * State of the tab list.
+   */
+  state: TabListState<unknown>
```

Comment:
only in the tab list not the top-level Tabs? also wonder which interactive states we want (e.g. focus within?).

Reply:
right anything were the collection is being sent from a child up to the parent, it'll be an issue
can we just remove the problematic data from state such that we can expose the rest?